<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="left_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>381</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>381</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="acc_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy and MSE</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
